package.json

```diff
@@ -6,19 +6,23 @@
   "main": "index.js",
   "scripts": {
     "dev": "webpack",
-    "deploy": "webpack --config webpack.production.config.js"
+    "deploy": "webpack --config webpack.production.config.js",
+    "test": "standard"
   },
   "license": "ISC",
   "devDependencies": {
     "babel-core": "^6.5.2",
-    "babel-eslint": "^5.0.0",
```

Comment:
babel-eslint shouldn't be required for non-bleeding edge `standard` parsing... I figured it was safe to remove this...